website/src/controllers/groups.js

```diff
@@ -837,7 +837,11 @@ questStart = function(req, res, next) {
     parallel.push(function(cb2){
       User.update({_id:m},updates,cb2);
     });
-  })
+
+    User.findOne({_id: m}, {pushDevices: 1}, function(err, user){
+      pushNotify.sendNotify(user, "HabitRPG", shared.i18n.t('questStarted') + ": "+ quest.text() );
```

Comment:
This should have a test for the following:

* that pushNotify _is called_ for user who has accepted a quest
* that pushNotify _is not called_ for user who has declined a quest
* that pushNotify _is not called_ for user who has a pending status on quest and quest was started anyway